js/ui/grid_core/ui.grid_core.editing_form_based.js

```diff
@@ -57,6 +57,14 @@ export const editingFormBasedModule = {
                     this._updateEditFormDeferred = null;
 
                     this.callBase.apply(this, arguments);
+
+                    this._rowsView.renderCompleted.add(() => {
```

Comment:
```suggestion
                    this._rowsView?.renderCompleted.add(() => {
```

tests are failed with `TypeError: Cannot read properties of undefined (reading 'renderCompleted')`, it seems that `_rowsView` isn't initialized in tests, so just add `?`